### PR TITLE
add new file Oct Email per bug 1407074

### PR DESCRIPTION
**Pull request by peiying2**
Polish only

### CODE REVIEW

File: en-US/emails/2017/FxnU_oct.lang [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+Hello healthier web.
+
+
+# Headline
+;“All browsers are the same, right?”
+“All browsers are the same, right?”
+
+
+;Not quite. Firefox is on a mission — to be faster and leaner than ever, always respectful of your privacy and open like no other browser out there. Because it’s built for people, not profit. Firefox is part of Mozilla’s mission to keep you in control of your online experience and ensure that the web stays weird, healthy and inviting to all.
+Not quite. Firefox is on a mission — to be faster and leaner than ever, always respectful of your privacy and open like no other browser out there. Because it’s built for people, not profit. Firefox is part of Mozilla’s mission to keep you in control of your online experience and ensure that the web stays weird, healthy and inviting to all.
```
Comment (peiying2) [inline en-US/emails/2017/FxnU_oct.lang]: I have to check what "weird" means here.

File: en-US/emails/2017/FxnU_oct.lang [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+;How? Because by “us,” we mean not just Firefox, but Mozilla, the technology nonprofit that backs us. Mozilla brings internet education to people all over the world, fights for your online rights and keeps corporate power in check.
+How? Because by “us,” we mean not just Firefox, but Mozilla, the technology nonprofit that backs us. Mozilla brings internet education to people all over the world, fights for your online rights and keeps corporate power in check.
+
+
+;You’re already doing good. And together we can do more. Learn how you can rock the power of your internet citizenship.
+You’re already doing good. And together we can do more. Learn how you can rock the power of your internet citizenship.
+
+
+;Thanks,
+Thanks,
```
Comment (flodolo) [inline en-US/emails/2017/FxnU_oct.lang]: It looks like these two strings are together, either add a comment or use one string: `Thanks,<br>The Firefox Team`.

File: en-US/emails/2017/FxnU_oct.lang [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+;We’re here to help you go your own way online and give you the power to browse freely, without being tracked or profited from.
+We’re here to help you go your own way online and give you the power to browse freely, without being tracked or profited from.
+
+
+;Because you are not a product. You’re an explorer on the open web. You deserve something different. Something better.
+Because you are not a product. You’re an explorer on the open web. You deserve something different. Something better.
+
+
+;We’ve got your back,
+We’ve got your back,
```
Comment (flodolo) [inline en-US/emails/2017/FxnU_oct.lang]: Why the comma? What's following?

File: en-US/emails/2017/FxnU_oct.lang [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+;You can even take your privacy with you. Firefox Focus for Android and iOS and make your digital footprint disappear with a single tap.
+You can even take your privacy with you. Firefox Focus for Android and iOS and make your digital footprint disappear with a single tap.
+
+
+;It’s a wild web out there — explore it with confidence.
+It’s a wild web out there — explore it with confidence.
+
+
+;Safe travels,
+Safe travels,
```
Comment (flodolo) [inline en-US/emails/2017/FxnU_oct.lang]: Same here

File: en-US/emails/2017/FxnU_oct.lang [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+;You’ve been using Firefox for a while now — maybe you’ve even made it your default browser. If you have, awesome! If not, here’s a recap on why you should: a) it’s faster; b) it’s private; c) it’s leaner than the big browsers; and, most importantly, d) it’s made for people, not profit.
+You’ve been using Firefox for a while now — maybe you’ve even made it your default browser. If you have, awesome! If not, here’s a recap on why you should: a) it’s faster; b) it’s private; c) it’s leaner than the big browsers; and, most importantly, d) it’s made for people, not profit.
+
+
+;No matter how you use Firefox, we’d love your feedback. Your input helps us tweak what’s not working for you and make what you’re already happy with even better. Everybody wins.
+No matter how you use Firefox, we’d love your feedback. Your input helps us tweak what’s not working for you and make what you’re already happy with even better. Everybody wins.
+
+
+;We’re all ears (and a tail),
+We’re all ears (and a tail),
```
Comment (flodolo) [inline en-US/emails/2017/FxnU_oct.lang]: Same here

File: en-US/emails/2017/FxnU_oct.lang (hunk 3 above)
Comment (peiying2) [reply en-US/emails/2017/FxnU_oct.lang]: This is the salute at the end of a message. We can add "<br> The Firefox Team with each of these messages.

File: en-US/emails/2017/FxnU_oct.lang (hunk 4 above)
Comment (peiying2) [reply en-US/emails/2017/FxnU_oct.lang]: same as above

File: en-US/emails/2017/FxnU_oct.lang (hunk 5 above)
Comment (peiying2) [reply en-US/emails/2017/FxnU_oct.lang]: same as above

File: en-US/emails/2017/FxnU_oct.lang (hunk 2 above)
Comment (peiying2) [reply en-US/emails/2017/FxnU_oct.lang]: If this is the solution, I will apply the fix to all 4 emails.

File: en-US/emails/2017/FxnU_oct.lang (hunk 3 above)
Comment (flodolo) [reply en-US/emails/2017/FxnU_oct.lang]: Makes sense, as long as people doing copy and paste understand what they're supposed to do.

File: en-US/emails/2017/FxnU_oct.lang [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+;You can even take your privacy with you. Firefox Focus for Android and iOS and make your digital footprint disappear with a single tap.
+You can even take your privacy with you. Firefox Focus for Android and iOS and make your digital footprint disappear with a single tap.
+
+
+;It’s a wild web out there — explore it with confidence.
+It’s a wild web out there — explore it with confidence.
+
+
+;Safe travels, <br> The Firefox Team
+Safe travels, <br> The Firefox Team
```
Comment (flodolo) [inline en-US/emails/2017/FxnU_oct.lang]: Remove the spaces around <br>, you don't want the new line starting with a leading space.

**Pull request merged**